modules/traefik/jobs/traefik.nomad

```diff
@@ -59,6 +63,15 @@ job "traefik" {
         }
       }
 
+      service {
+        name = "prometheus-client"
```

Comment:
Since this is registered globally into Consul, I would suggest naming it something like `traefik-prometheus` to indicate where it comes from.

Otherwise looking at the Consul page, one won't be able to easily tell `prometheus-client` comes from which service.

Reply:
All the services that expose a prometheus client for the prometheus server should be lumped under the service `prometheus-client`
https://github.com/dsaidgovsg/terraform-modules/blob/master/modules/prometheus/packer/ami/site.yml#L75-L120
Services named as `prometheus-client` are meant to be differentiated by their tags. Otherwise, we would have to rebuild the Prometheus AMI to add a new service name to the scrape config every time we wanted to instrument a new incoming service.

Reply:
I see, so the idea is the opposite, to lump all the exposed prometheus ports under a same service name. :+1: